numba/core/config.py

```diff
@@ -154,6 +154,11 @@ def optional_str(x):
             "NUMBA_ALWAYS_WARN_UNINIT_VAR", int, 0,
         )
 
+        # Whether to warn about kernel launches where the grid size will
+        # underutilize the GPU due to low occupancy. On by default.
+        LOW_OCCUPANCY_WARNINGS = _readenv(
```

Comment:
```suggestion
        CUDA_LOW_OCCUPANCY_WARNINGS = _readenv(
```
All other CUDA related config vars start with `CUDA_` or have `CUDA` in their name, suggest calling this something like the above.

Reply:
Done.